FAQ.md

```diff
@@ -313,4 +313,21 @@ If you are in China,docker container uses UTC time(Coordinated Universal Time) a
 
 ## How to join Dragonfly as a member
 
-Please check the [CONTRIBUTING.md](CONTRIBUTING.md#join-dragonfly-as-a-member)
\ No newline at end of file
+Please check the [CONTRIBUTING.md](CONTRIBUTING.md#join-dragonfly-as-a-member)
+
+## Can I config the gc parameter
+
+Yes, we can config the supernode gc parameter.
+
+There are some parameters we can config in supernode.
+
+Name                           | Default Value | Description
+------------------------------ | ------------- | ----------
+gc-initial-delay               | 6s            | it is the delay time from the start to the first GC execution.
+gc-meta-interval               | 2min          | it is the interval time to execute the GC meta.
+task-expire-time               | 3min          | when a task is not accessed within the time,it will be treated to be expired.
+peer-gc-delay                  | 3min          | it is the delay time to execute the GC after the peer has reported the offline.
+
+In supernode ,gc will begin `gc-initial-delay` time after supernode works.Then supernode will run peer-gc goroutine and task-gc goroutine every  `gc-meta-interval` time.
+If a task isn't accessed by dfgets in `task-expire-time` time, task-gc goroutine will gc this task, but now supernode only gc the task information in memory.
+If a peer reports it servers down, peer-gc goroutine will gc this peer after `peer-gc-delay` time.
```

Comment:
`If a peer reports it servers down` should be more clear. Actually I cannot quite follow it.

Reply:
I agree with you,I will fix it

Reply:
done